packages/ember-glimmer/tests/utils/shared-conditional-tests.js

```diff
@@ -407,33 +462,41 @@ export class SharedConditionalsTest extends AbstractConditionalsTest {
 
     this.assertText('T1F2');
   }
+}
 
-  ['@test it maintains DOM stability when condition changes from a truthy to a different truthy value']() {
-    this.renderValues(true);
-
-    this.assertText('T1');
+export class SharedSyntaxConditionalsTest extends SharedConditionalsTest {
+  ['@htmlbars it does not update when the unbound helper is used']() {
```

Comment:
Not to self: this is `@htmlbars` for now because we haven't implemented `(unbound` for glimmer. I wish we had real method decorators so we could write something like `@skip("glimmer", "unbound is not yet implemented")`. Maybe we should make it a rule for now that we should add a comment above each `@htmlbars` and `@glimmer`?